src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3515,31 +3515,146 @@ public void ConstantInterpolatedStringsError()
             string source = @"
 class C
 {
-    void M()
+    void M(string ParamDefault = ""Academy City"")
```

Comment:
>""Academy City"" [](start = 33, length = 16)

This is not what I meant by parameter default. I meant using an interpolated string in this location. Please break that out into a separate test.

Reply:
Ah, that makes sense. Broke out a new test for interpolated string default, and also tested for use of constant in switch statement.

Reply:
We need to test langversion for these locations :). Sorry